Bottom: 8ad44f2990b35cb444d49d0eadb14789d6eac1e4
Top:    037021a01cafa5aac2e930da6b981f6f8fbfa64c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-10 03:28:50 +0000

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index b0a5049..35fc720 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -150,8 +150,7 @@ draw_grid ()
   if (n > npoints)
     {
       npoints = n + 10;
-      points =
-        MyRealloc (points, npoints * 2 * sizeof (GLfloat), "gtk_draw_grid");
+      points = realloc (points, npoints * 2 * sizeof (GLfloat));
     }
 
   glEnableClientState (GL_VERTEX_ARRAY);
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 89ee1e8..e75d36b 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -186,8 +186,7 @@ ghid_draw_grid ()
   if (n > npoints)
     {
       npoints = n + 10;
-      points =
-	MyRealloc (points, npoints * 2 * sizeof (GLfloat), "gtk_draw_grid");
+      points = realloc (points, npoints * 2 * sizeof (GLfloat));
     }
 
   glEnableClientState (GL_VERTEX_ARRAY);
